{"name": "test_div_int[4-3-1.333333333333333]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.3333333333333333 == 1.333333333333333\n  +1.3333333333333333\n  -1.333333333333333", "trace": "self = <HomeWork.P_20201027.tests.test_calc.TestCalc object at 0x000002076172B970>\na = 4, b = 3, c = 1.333333333333333\n\n    @allure.story(\"两个正整数相除\")\n    @pytest.mark.parametrize(\"a, b, c\", [\n        [2, 2, 1],\n        [4, 3, 1.333333333333333],\n        [3, 6, 0.5]\n    ])\n    def test_div_int(self, a, b, c):\n>       assert self.calc.div(a, b) == c\nE       assert 1.3333333333333333 == 1.333333333333333\nE         +1.3333333333333333\nE         -1.333333333333333\n\ntests\\test_calc.py:49: AssertionError"}, "parameters": [{"name": "a", "value": "4"}, {"name": "b", "value": "3"}, {"name": "c", "value": "1.333333333333333"}], "start": 1603925659273, "stop": 1603925659274, "uuid": "0adb58c8-0984-4a6e-aaed-eff7c81f2bcc", "historyId": "160e0ae4e5f54072aeb46e41b69f90ce", "testCaseId": "7aa9e8abb1dc46233d6ed4a9fb15d5a8", "fullName": "tests.test_calc.TestCalc#test_div_int", "labels": [{"name": "feature", "value": "calc.py模块测试用例"}, {"name": "story", "value": "两个正整数相除"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-UA4RMQR"}, {"name": "thread", "value": "8744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calc"}]}